inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/clickhouse/DefaultEventHandler.java

```diff
@@ -0,0 +1,333 @@
+
+    private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * parse
+     * @param idConfig
+     * @param event
+     * @return
+     */
+    @Override
+    public Map<String, String> parse(ClickHouseIdConfig idConfig, ProfileEvent event) {
+        final Map<String, String> resultMap = new HashMap<>();
+        // parse fields
+        String delimeter = idConfig.getSeparator();
+        char cDelimeter = delimeter.charAt(0);
+        String strContext = null;
+        // for tab separator
+        byte[] bodyBytes = event.getBody();
+        int msgLength = event.getBody().length;
+        int contentOffset = idConfig.getContentOffset();
+        if (contentOffset > 0 && msgLength >= 1) {
+            strContext = new String(bodyBytes, contentOffset, msgLength - contentOffset, Charset.defaultCharset());
+        } else {
+            strContext = new String(bodyBytes, Charset.defaultCharset());
+        }
+        // unescape
+        List<String> columnValues = UnescapeHelper.toFiledList(strContext, cDelimeter);
+        // column size
+        List<String> contentFieldList = idConfig.getContentFieldList();
+        int matchSize = Math.min(contentFieldList.size(), columnValues.size());
+        for (int i = 0; i < matchSize; i++) {
+            resultMap.put(contentFieldList.get(i), columnValues.get(i));
+        }
+
+        // ftime
+        String ftime = dateFormat.format(new Date(event.getRawLogTime()));
+        resultMap.put("ftime", ftime);
+        // extinfo
+        String extinfo = getExtInfo(event);
+        resultMap.put("extinfo", extinfo);
+        return resultMap;
+    }
+
+    /**
+     * getExtInfo
+     * 
+     * @param  event
+     * @return
+     */
+    public static String getExtInfo(ProfileEvent event) {
+        String extinfoValue = event.getHeaders().get(KEY_EXTINFO);
+        if (extinfoValue != null) {
+            return KEY_EXTINFO + "=" + extinfoValue;
+        }
+        extinfoValue = KEY_EXTINFO + "=" + event.getHeaders().get(EventConstants.HEADER_KEY_SOURCE_IP);
+        return extinfoValue;
+    }
+
+    /**
+     * setValue
+     * @param idConfig
+     * @param columnValueMap
+     * @param pstat
+     * @throws SQLException 
+     */
+    public void setValue(ClickHouseIdConfig idConfig, Map<String, String> columnValueMap, PreparedStatement pstat)
+            throws SQLException {
+        List<Pair<String, Integer>> dbFieldList = idConfig.getDbFieldList();
+        for (int i = 1; i <= dbFieldList.size(); i++) {
+            Pair<String, Integer> pair = dbFieldList.get(i - 1);
+            String fieldValue = columnValueMap.getOrDefault(pair.getKey(), "");
+            int fieldType = pair.getValue();
+            switch (fieldType) {
+                case Types.ARRAY :
+                    // ARRAY = 2003;
+                    // pstat.setArray(1, null); // String
+                    pstat.setString(i, fieldValue);
```

Comment:
There are so many `pstat.setString(i, fieldValue);` in some different cases, suggest change as following:

```java
case A:
case B:
case C:
    pstat.setString(i, fieldValue);
    break;
case D:
    // ...
```

Reply:
fix it